zScanner/ViewModels/NewDocument/NewDocumentTypeViewModel.swift

```diff
@@ -51,7 +43,6 @@ class NewDocumentTypeViewModel {
             return [
                 ListPickerField<DocumentTypeDomainModel>(title: "form.listPicker.title".localized, list: documentTypes),
                 TextInputField(title: "form.documentDecription.title".localized, validator: { _ in true }),
-                DateTimePickerField(title: "form.dateTimePicker.title".localized, validator: { $0 != nil }),
```

Comment:
Keep all the logic on the page, just remove the field for date time picker. 

We want as minimum changes as possible from the original codebase. So we can easier merge them in the future if necessary.